cm2350/e200z7.py

```diff
@@ -395,7 +396,7 @@ def reset(self):
         self.systimeReset()
         for key, module in self.modules.items():
             if hasattr(module, 'reset'):
-                logger.debug("reset: Resetting %r...", key)
+                logger.debug("reset: Resetting %s...", key)
                 module.reset(self)
```

Comment:
Same thing here.  The base module class should implement a NOP reset function that gets overridden by subclasses.

Reply:
Similar comment from the previous one. how would you prefer to handle this? We can probably add a check during "module install" that ensures the necessary functions are there?

Reply:
updated when the `Module` changes were made